test/controllers/DisableLoadingPartialsSpec.scala

```diff
@@ -0,0 +1,112 @@
+
+import config.{AppConfig, CFConfig}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.mockito.MockitoSugar.mock
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.Application
+import play.api.http.Status.GONE
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.mvc.Result
+import play.api.test.FakeRequest
+import play.api.test.Helpers.{contentAsString, defaultAwaitTimeout, status}
+import uk.gov.hmrc.auth.core.AuthConnector
+import uk.gov.hmrc.http.HttpClient
+import util.BackUrlValidator
+
+import scala.concurrent.Future
+
+class DisableLoadingPartialsSpec extends AnyWordSpec with GuiceOneAppPerSuite with Matchers {
+
+  override def fakeApplication(): Application =
+    new GuiceApplicationBuilder()
+      .configure(
+        "metrics.jvm"      -> false,
+        "metrics.enabled"  -> false,
+        "auditing.enabled" -> false,
+        "disablePartials"  -> true
+      )
+      .bindings(bind[AppConfig].to[CFConfig])
+      .bindings(bind[AuthConnector].to(mock[AuthConnector]))
+      .bindings(bind[HttpClient].to(mock[HttpClient]))
+      .bindings(bind[BackUrlValidator].to(mock[BackUrlValidator]))
+      .build()
+
+  "ReportProblemControllerSpec" should {
+    "returns 410 when partialIndex is called " in new TestScope {
+      val result: Future[Result] = mockReportProblemController.partialIndex(
+        preferredCsrfToken = None,
+        service = Some(serviceName)
+      )(FakeRequest())
+
+      status(result)          should be(GONE)
+      contentAsString(result) should be(expectedResponseMessage)
+    }
+
+    "returns 410 when partialAjaxIndex is called" in new TestScope {
+      val result: Future[Result] = mockReportProblemController.partialAjaxIndex(
+        service = Some(serviceName)
+      )(FakeRequest())
+
+      status(result)          should be(GONE)
+      contentAsString(result) should be(expectedResponseMessage)
+    }
+  }
+
+  "ContactHmrcController" should {
+    "returns 410 when partialIndex is called" in new TestScope {
+      val result: Future[Result] = mockContactHmrcController.partialIndex(
+        submitUrl = "",
+        csrfToken = "",
+        service = Some(serviceName),
+        renderFormOnly = true
+      )(FakeRequest())
+
+      status(result)          should be(GONE)
+      contentAsString(result) should be(expectedResponseMessage)
+    }
+  }
+
+  "FeedbackController" should {
+    "returns 410 when partialIndex is called" in new TestScope {
+      val result: Future[Result] = mockFeedbackController.partialIndex(
+        submitUrl = "",
+        csrfToken = "",
+        service = Some(serviceName),
+        referer = None,
+        canOmitComments = true,
+        referrerUrl = None
+      )(FakeRequest())
+
+      status(result)          should be(GONE)
+      contentAsString(result) should be(expectedResponseMessage)
+    }
+  }
+
+  trait TestScope {
```

Comment:
total knitpick on this but I think we shouldn't call it a mockFeedbackController because it's using mocks but not really a mock itself, we could drop the TestScope trait altogether and just `val result: Future[Result] = app.injector.instanceOf[FeedbackController].partialIndex(...`